hal_flir_d46:
  ros__parameters:
    port: /dev/ttyS1
    limits_enabled: True

    baud: 9600
    default_velocity: 0.1
    hz: 10

    # Ranges for Pan/Tilt. If not set, min/max values
    # queried from hardware are set.

    #min_tilt: 
    #max_tilt: 
    #min_tilt_speed: 
    #max_tilt_speed: 
    #tilt_step: 
    #min_pan: 
    #max_pan: 
    #min_pan_speed: 
    #max_pan_speed: 
    #pan_step: 

    publishers:
      state: /ptu/state

    services:
      set_pan: /ptu/set_pan
      set_tilt: /ptu/set_tilt
      set_pantilt: /ptu/set_pan_tilt
      set_pantilt_speed: /ptu/set_pantilt_speed
      reset: /ptu/reset
      get_limits: /ptu/get_limits

    actions: 
      set_pan: /ptu/set_pan
      set_tilt: /ptu/set_tilt
      set_pantilt: /ptu/set_pan_tilt